frontend/src/features/plots/PlotContainer.tsx

```diff
@@ -202,7 +203,7 @@ const useExtractedData = (
     }
 
     const newEntries = Object.entries(extracted)
-      .filter(([run]) => !plotData.hasOwnProperty(run))
+      .filter(([run]) => !(run in plotData))
```

Comment:
The same applies as before.

Reply:
Similar: we assume that this is a pure Redux object so it doesn't need to be restrictive.